abis/linux/wait.h

```diff
@@ -22,4 +22,7 @@
 #define WIFCONTINUED(x) ((x) == 0xffff)
 #define WCOREDUMP(x) ((x) & WCOREFLAG)
 
+// glibc extension, but also useful for the kernel
```

Comment:
Use C-style comments in public facing headers (yes, I'm aware of the line comment right below it, I'll be replacing them all at some point in the future).

Reply:
Why don't we merge this and change them later, then?

Reply:
Adding more serves no good other than to increase the size of further patches (hence increasing the grounds of conflict)